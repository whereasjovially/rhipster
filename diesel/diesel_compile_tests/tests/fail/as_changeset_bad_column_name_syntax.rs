#[macro_use]
extern crate diesel;

table! {
    users {
        id -> Integer,
        name -> Text,
    }
}

#[derive(AsChangeset)]
#[table_name = "users"]
struct User {
    #[column_name]
    name: String,
}

fn main() {}
