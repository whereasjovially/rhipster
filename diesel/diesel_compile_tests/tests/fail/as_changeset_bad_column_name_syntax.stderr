error: `column_name` must be in the form `column_name = "value"`
  --> $DIR/as_changeset_bad_column_name_syntax.rs:14:7
   |
14 |     #[column_name]
   |       ^^^^^^^^^^^

error[E0412]: cannot find type `column_name` in module `users`
  --> $DIR/as_changeset_bad_column_name_syntax.rs:14:7
   |
14 |     #[column_name]
   |       ^^^^^^^^^^^ not found in `users`

error[E0425]: cannot find value `column_name` in module `users`
  --> $DIR/as_changeset_bad_column_name_syntax.rs:14:7
   |
14 |     #[column_name]
   |       ^^^^^^^^^^^ not found in `users`
